### ISSUE TITLE
Hybrid numbering

### ISSUE THREAD

**Issue by Konfekt** [matf/relativenumberruler#3]
How about hybrid line numbering as in vim 7.4 with

```
set number relativenumber
```

Then the cursor line has the absolute line number instead of 0, and the lines above and below relative numbers.

For this, a simple check at

```
String lineStr = Integer.toString(Math.abs(currentLine - modelLine));
```
should suffice.

**Comment by ashwin** [comment]
+1

I use this hybrid numbering in Vim and would love to see it in Eclipse too :-)

**Comment by matf** [comment]
Fixed in 1.1.0. See http://matf.github.io/relativenumberruler/ for details. Enjoy.

**Comment by Konfekt** [comment]
Cool, thanks!
